README.md

```diff
@@ -45,9 +45,41 @@ Note: The version tested with Zivid cameras is 2.0.0.
 
 2. [**Download Zivid Sample Data**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/450363393/Sample+Data).
 
-3. Launch Visual Studio 2017.
 
-4. Open and run one of the samples.
+### Windows
+
+Launch the Command Prompt by pressing *Win + R* keys on the keyboard, then type cmd and press Enter.
+
+Navigate to a location where you want to clone the repository, then run to following command:
+
+```bash
+git clone https://github.com/zivid/zivid-csharp-samples
+```
+
+[comment]: <> (Choose a sample solution and configure it with CMake.)
+[comment]: <> (Launch Visual Studio, open, build, and run the sample solution.)
+
+Open *ZividNETSamples.sln* in Visual Studio, build it, and run it. If you are uncertain about doing this, check out our [**tutorials for configuring and building C# Samples**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/534708242).
+
+### Ubuntu
+
+Open the Terminal by pressing *Ctrl + Alt + T* keys on the keyboard.
+
+Navigate to a location where you want to clone the repository, then run to following commands:
+
+```bash
+git clone https://github.com/zivid/zivid-csharp-samples
+cd zivid-csharp-samples
+```
+Open *ZividNETSamples.sln* in Visual Studio, build it, and run one if the samples.
+
+## External libaries 
+Some of the samples depend on external libraries, in particular *YamlDotNet* and *MathNet.Numerics*. Both of these libraries can be installed using NuGet package manager within Visual Studio. 
+
+To install the libraries:
+- **YamlDotNet**: Type `PM> Install-Package YamlDotNet` in the Package Manager Console.           
+- **MathNet.Numerics**: Type `PM> Install-Package MathNet.Numerics` in the Package Manager Console.
```

Comment:
This is nice but in converges from what we have for C++ repo. In C++ repo, we just have links to the dependencies, the instructions on how to install them are on the KB. For C#, you've added the instructions here. I think we should be consistent.

Reply:
(reffering to Anders comment), Using NuGet packet manager this is done automatically, hence we don't need any explanation on how to download these libraries. After I have made NuGet work/understand how it works, I can create some instructions on KB (I saw that we don't have instructions on how to install and build Zivid samples in Ubuntu either, so can create that as well).

Reply:
I don't think C#/.NET works on Ubuntu

Reply:
But do add instructions on KB for packages if needed

Reply:
There should be no documentation needed. If installing these requirements require any manual effort from the user, we're doing it wrong.

Reply:
Done, removed all Ubuntu related stuff and updated the KB article on how to install third-party libraries; https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/534708242/Build+C+Samples+using+Visual+Studio

Reply:
LGTM, but leaving it to @SatjaSivcev to resolve since it's his thread.

Reply:
https://support.zivid.com/latest/academy/samples/c-sharp/build-c-sharp-samples-using-visual-studio.html